### PR TITLE
Added LimitedLengthString type to avoid log errors

### PR DESCRIPTION
**Pull request by bakspace-itk**

### CODE REVIEW

**Review by ghbm-itk — CHANGES_REQUESTED** [review]
I would like some tests of this added to the automated tests

**Comment by bakspace-itk** [comment]
Pull request was closed when issue was removed from release.

File: changelog.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - 'Pausing' status added to triggers.
 - Disabling a 'Running' trigger in Orchestrator will set its status to 'Pausing'.
 - Scheduler will change a 'Pausing' trigger to 'Paused' when the process is done.
+- Added truncated string method for limiting long logs
```
Comment (ghbm-itk) [inline changelog.md:16]: I think this should be under 'Changed'.
And it should be more specific: "Log messages over 8000 characters are now truncated preserving the beginning and end"

File: changelog.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Sort triggers & queues by name by default.
+- Log messages over 8000 characters are now truncated preserving the beginning and end
```
Comment (ghbm-itk) [inline changelog.md:20]:
```suggestion
- Log messages over 8000 characters are now truncated preserving the beginning and end.
```

**Review by ghbm-itk — CHANGES_REQUESTED** [review]

**Review by ghbm-itk — APPROVED** [review]

**Review by ghbm-itk — APPROVED** [review]

**Pull request merged**